tests/mainnet/beacon_chain/light_client/optimistic_update.yaml

```diff
@@ -0,0 +1,3 @@
+# Optimistic update for slot 6718464
```

Comment:
Might want to clarify that this is the `signature_slot` as you did in the updates.yaml

Reply:
Done